src/wrapped/wrappedlibltdl7_private.h

```diff
@@ -2,28 +2,15 @@
 error Meh...
 #endif
 
-GO(lt__slist_box, pFp)
```

Comment:
@ptitSeb I think this library is dangerous to wrap. It seems a wrapper on dlopen and stuff.

Libtool provides a small library, called libltdl, that aims at hiding the various difficulties of dlopening libraries from programmers.

Reply:
Right, I'll delete this wrapped.

Reply:
Yeah, wrapping on top of libdl might be tricky. It can be usefull, if it can be avoided, that would be better.
If the libs is small, we can provide the x86_64 version with box64 if it helps...